src/os/bluestore/BlueStore.cc

```diff
@@ -6210,6 +6211,9 @@ void BlueStore::_txc_state_proc(TransContext *txc)
 
     case TransContext::STATE_IO_DONE:
       //assert(txc->osr->qlock.is_locked());  // see _txc_finish_io
+      if (txc->had_aios) {
```

Comment:
I think build error is from  this line.
There is an typo
The variable name is had_ios, not had_aios.

Thank you!